modules/ting/ting.client.inc

```diff
@@ -1,777 +1,112 @@
 <?php
 /**
  * @file
- * Wrapper functions for Ting client.
- */
-
-/**
- * Used to indicate that a given cache entry have return no reply from the data
- * well.
+ * Legacy ting functions.
```

Comment:
Would it be an idea to add a "secret" variable that could be set in settings.php to log usage of these deprecated functions to help all developers in the transition to the new providers? This should only be used in development mode and/or on tests sites.